pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -746,7 +828,7 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *v1.Serv
 				_, err := v2pools.CreateMember(lbaas.network, pool.ID, v2pools.CreateMemberOpts{
 					ProtocolPort: int(port.NodePort),
 					Address:      addr,
-					SubnetID:     lbaas.opts.SubnetId,
+					SubnetID:     getStringFromServiceAnnotation(apiService, ServiceAnnotationLoadBalancerSubnetID, lbaas.opts.SubnetId, lbaas.opts.EnableAnnotationSubnetId),
```

Comment:
We have supported for autoprobing subnet-id, please do a rebase for it, get more info from Line 609.